lib/src/loaders.dart

```diff
@@ -0,0 +1,213 @@
+import 'dart:convert' show utf8;
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/src/utilities/http.dart';
+import 'package:vector_graphics/vector_graphics.dart';
+import 'package:vector_graphics_compiler/vector_graphics_compiler.dart';
+
+/// Compute will mess up tests because of FakeAsync.
+final bool _isTest = Platform.executable.endsWith('flutter_tester') ||
```

Comment:
We should guard this with a kDebugMode check

Reply:
and probably a kIsWeb check too

Reply:
Why kIsWeb?

Reply:
Platform.whatever aren't supported